keras_cv/models/object_detection/retinanet/retinanet.py

```diff
@@ -119,6 +120,10 @@ class RetinaNet(Task):
             default `prediction_decoder` layer is a
             `keras_cv.layers.MultiClassNonMaxSuppression` layer, which uses
             a Non-Max Suppression for box pruning.
+        feature_pyramid: (Optional) A `keras.layers.Layer` that produces
+            a list of 4D feature maps (batch dimension included)
+            when called on the `backbone`. If not provided, the reference
```

Comment:
When called on the pyramid-level outputs of the backbone

Reply:
Not sure I follow -- the code looks like 
```
backbone_outputs = feature_extractor(images)
features = feature_pyramid(backbone_outputs)
```

I don't really know what it means for a layer to be called on another layer -- it's called on the output of the other layer

Reply:
Yes, you are right! I was thinking of the original design where the FPN also handled feature extraction. I think it's OK to standardize on `pyramid_level_inputs`. @gianlucasama can you update the docstring?